src/Controls/tests/TestCases.HostApp/Issues/Issue27627.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 27627, "[Windows] Border Automation Peer", PlatformAffected.UWP)]
+public partial class Issue27627 : ContentPage
+{
+	public Issue27627()
+	{
+		var grid = new Grid();
+
+		var border = new Border
```

Comment:
Could expand the sample to test more scenarios?

- Nested Borders. Each gets own automation peer via `OnCreateAutomationPeer()`
- Border with custom Content. `IsContentElementCore = true` ensures nested content exposed to screen readers